UPDATING.md

```diff
@@ -24,6 +24,7 @@ assists people when migrating to a new version.
 
 ## Next
 
+- [23186](https://github.com/apache/superset/pull/23186): Superset will refuse to start if a default `SECRET_KEY` is detected on a non Flask debug setting.
```

Comment:
This looks like a potentially breaking change. If we want to put this in a patch or minor release, can we issue a warning first?

Reply:
a warning to the community? I think so, changing your default secret_key is really mandatory